source/languageHandler.py

```diff
@@ -133,7 +133,7 @@ def pgettext(context, message):
 				return message
 	else:

```

Comment:
Why do we have this else statement? Shouldn't this fail if translations  is not an instance of gettext.GNUTranslations? If not, what else do we expect and how should that be handled?

Reply:
I guess it means that upon failure it falls back to the original message, eg English. I think a log message is a good idea, and if the log message isn't enough to explain it, some comments too.